src/views/Playground.vue

```diff
@@ -15,11 +17,37 @@ const defaultParams = {
 
 const route = useRoute();
 const { strategies } = useApp();
+const { notify } = useNotifications();
+const { t } = useI18n();
 
 let provider;
 
 const strategy = computed(() => strategies.value[route.params.name]);
-const strategyExample = computed(() => strategy.value.examples?.[0]);
+const strategyExample = computed(() => {
+  const queryParamsIndex = window.location.hash.indexOf('?query=');
```

Comment:
You shouldn't use window location here, you can get the url query params with Vue router directly